Feat: Animated SVG vue components support

maheshwarano4s opened the pull request:
**Task:**
[Dashboard V2 | Fix Loader](https://www.notion.so/o4s/Dashboard-V2-Fix-Loader-384dc405c34842b29add223270d48298)

**Feature and Changes:**
1. Added support for O4SLoader using o4s-svg library by introducing animated svg concept.

Kasahs commented:
Again very large diffs for seemingly small change @maheshwarano4s please justify.

Kasahs reviewed (changes requested):
Very large diff for seemingly simple change. Needs justification.

himanko4s commented on package.json:
```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
-    "@original4sure/o4s-ui": "file:../o4s-ui",
+    "@original4sure/o4s-ui": "^1.1.7",
```
is there any need to update this from latest lib version? @maheshwarano4s

maheshwarano4s replied:
Yes we have to update, that will be good pratics.
But 1.1.7 will be good enough for o4s-svg. if we need new component which is not part of 1.1.7 release then we have to update otherwise it's fine.

himanko4s reviewed (approved):
LGTM

maheshwarano4s commented:
we introduced o4s-icon and o4s-svg library for reuse purpose and we don't want to keep any icon or svg asset in application side so that we use it in many application we want with the same code base as lib.
the reason why we move to svg asset as lib, icon font had as limitation of 6000 unique value in UTF-16 private code space. and we are already supporting two icon library, one is our own and primevue icons. if we add more icons as font we could be run out of code and converting svg to font and validating code not an easy task when icon size grows.
So create o4s-svg library, Again we categorized into two type as icons and vectors.
Icons -> have some rules like size, svg type.
vector -> means any valid svg for UI screen pages.
and which support our design pattern.
when we find loader changes we see animation, also we have to ensure our design guideline as well. If we use gif type then it could break with our design standard.
So create another category in o4s-svg library as animated vectors, designed gave only svg, where we added animation to it. and exported in lib.
To order to use o4s-svg lib in application we have to give docs for developer. we deployed two website for this.
SVG Lib Docs - https://original4sure.github.io/o4s-svg/dist/
icon lib Doc - https://original4sure.github.io/o4sicons/dist/ft/o4sicons
Based on all the above requirement made changes to respective files.

Kasahs commented:
Want to understand this better let's discuss in next week's design call.
